config/config-feature-flags.yaml

```diff
@@ -29,6 +29,8 @@ data:
   # See more in the Affinity Assistant documentation
   # https://github.com/tektoncd/pipeline/blob/main/docs/affinityassistants.md
   # or https://github.com/tektoncd/pipeline/pull/2630 for more info.
+  #
+  # Note: This feature flag is deprecated and will be removed in 9 months. Consider using `coschedule` feature flag to configure Affinity Assistant behavior.
```

Comment:
Can you update line 46, where it says that coscheduling is non functional?

Reply:
Oh, sorry I thought I have removed in the last implementation PR... Removed!